bin/studio-cli.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# The least terrible way to resolve a symlink to its real path.
+function realpath() {
+  /usr/bin/perl -e "use Cwd;print Cwd::abs_path(@ARGV[0])" "$0";
+}
+
+CONTENTS="$(command dirname "$(command dirname "$(command dirname "$(realpath "$0")")")")"
```

Comment:
This does not look like "the least terrible way" to resolve three parent directories above the real path 😆 Never mind, though. Sticking with that works for GitHub Desktop seems like a way to avoid problems.